web/gds-user-ui/src/components/ErrorComponent/RequiredElementMissing.tsx

```diff
@@ -1,16 +1,55 @@
-import { Alert, AlertIcon } from '@chakra-ui/react';
+import {
+  Alert,
+  AlertIcon,
+  ListItem,
+  UnorderedList,
+  AlertDescription,
+  Box,
+  Text,
+  HStack
+} from '@chakra-ui/react';
 import { Trans } from '@lingui/macro';
 interface RequiredElementMissingProps {
   elementKey?: number;
+  errorFields?: any[];
 }
 
-const RequiredElementMissing = ({ elementKey }: RequiredElementMissingProps) => {
+const RequiredElementMissing = ({ elementKey, errorFields }: RequiredElementMissingProps) => {
   console.log('k', elementKey);
 
   return (
-    <Alert status="error" borderRadius="lg" my={4}>
-      <AlertIcon />
-      <Trans>There is an error or missing data in this section. Please edit the section.</Trans>
+    <Alert
+      status="error"
+      flexDirection="column"
+      textAlign="left"
+      alignItems="left"
+      justifyContent="left"
+      variant="subtle"
+      borderRadius="lg"
+      my={4}>
+      <HStack>
+        <AlertIcon />
+        <Trans>There is an error or missing data in this section. Please edit the section.</Trans>
+      </HStack>
+
+      <Box ml="35" mt="2">
+        <AlertDescription>
+          <UnorderedList>
+            {errorFields?.map((errorField: any, index) => {
+              console.log('errorField', errorField);
+              return (
+                <ListItem key={index}>
+                  <Text as="span" fontWeight="bold">
+                    {errorField?.field}
+                    {': '}
+                  </Text>
+                  <Text as="span">{errorField?.error}</Text>
```

Comment:
Should we ask the backend to provide more user friendly error messages here if this option is taken instead of highlighting the missing field in red like in the Figma design?

<img width="1195" alt="Screenshot 2023-05-29 at 9 38 57 AM" src="https://github.com/trisacrypto/directory/assets/94616884/90f0d550-f596-4cef-87f7-9792785421ee">

Reply:
@pdeziel what do you think about reformatting the error message?